packages/documentation/src/stories/components/back-to-top/back-to-top.stories.ts

```diff
@@ -0,0 +1,55 @@
+import { StoryContext, StoryFn, StoryObj } from '@storybook/web-components';
+import { MetaComponent } from '@root/types';
+import { html } from 'lit';
+
+const meta: MetaComponent = {
+  id: '1a1b4cab-d0a8-4b01-bd85-b70e18668cb5',
+  title: 'Components/Back To Top Button',
+  component: 'post-back-to-top',
+  tags: ['package:WebComponents'],
+  parameters: {
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/design/JIT5AdGYqv6bDRpfBPV8XR/Foundations-%26-Components-Next-Level?node-id=18-11',
+    },
+  },
+};
+
+export default meta;
+
+type Story = StoryObj;
+
+export const Default: Story = {
+  decorators: [
```

Comment:
Can you move the decorator and renderer to the meta and reuse it for the snapshot to avoid code duplication?